ptsprojects/zephyr/sm_wid.py

```diff
@@ -112,6 +113,51 @@ def hdl_wid_143(desc):
 
     return True
 
+
+def hdl_wid_145(desc):
+    """
+    Please configure IUT's OOB data flag with 'No remote OOB data present'
+
+    TODO: This is done by default but we should set it explicitly
+    """
+    return True
+
+
+def hdl_wid_146(desc):
+    """
+    Please configure IUT's OOB flag with 'Remote OOB data present'
+
+    TODO: The flag will be set when we handle wid 149 - set remote oob data
+    """
+    return True
+
+
+def hdl_wid_147(desc):
+    """
```

Comment:
{147,%s,SM}Please enter 16 bytes IUT's OOB Data (confirmation).
{148,%s,SM}Please enter 16 bytes IUT's OOB Key (random number).

Assumption 1) R and C can not be used alone and they are always required together. 
Assumption 2) OOB sc local data is fixed.
With assumption 1 and 2, should we call this function two times? We can save execution time by calling it once and set r and c locally. If you want to keep it this way I can understand for integrity of the wid implementation.

Reply:
I agree, it's a bit confusing why they have 2 different wids for the OOB.
I just copied over the implementation from mynewt, maybe we later find out that there is a reason for it.

For the time being, how about just adding a TODO comment?

Reply:
or not :)

Reply:
for now lets leave it like this